### ISSUE TITLE
Problem with constant when installing globally

### ISSUE THREAD

**Issue by marioquartz** [composer-unused/composer-unused-plugin#7]
I try to install globally using "`composer global require composer-unused/composer-unused-plugin`"
And after enable the trust there are an ErrorException:
```
[ErrorException]                                                             
  Use of undefined constant UNUSED_COMPOSER_INSTALL - assumed 'UNUSED_COMPOSE  
  R_INSTALL' (this will throw an Error in a future version of PHP)
```

**Comment by icanhazstring** [comment]
Hm. The error is not very helpful.
I receive it as well. Need to investigate further why this error is produced.
Since the only two places, this is used. But I think I know where the problem is coming from.

The problem might originate from here:
https://github.com/composer-unused/composer-unused/blob/main/config/services.php#L34

Either way. Thanks for the report. I will dig into this :)

**Comment by rooselle** [comment]
I also have this error when installing inside project with `composer require composer-unused/composer-unused-plugin`. When asked `Do you trust "composer-unused/composer-unused-plugin" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?]` I type `y` and get following error:
```
 Install of composer-unused/composer-unused-plugin failed

In services.php line 32:
                                                
  [Error]                                       
  Undefined constant "UNUSED_COMPOSER_INSTALL" 
```

Thanks!

**Comment by icanhazstring** [comment]
Thanks for the report you two. Let me take a look at this 👍

**Comment by theofidry** [comment]
The problem is easy to reproduce. My gut feeling is that the clauses in `bin/unused` where you initialize that constant are not behaving as you expect within a PHAR

**Comment by icanhazstring** [comment]
Yep. The plugin currently needs some love. At the moment the plugin is not utilizing the phar to 100%. That is why the plugin currently has a direct dependency to composer-unused.

This will be removed in a future release. Hopefully this will also remove this problem.

**Comment by icanhazstring** [comment]
Closed in favor of #16
